models.py

```diff
@@ -25,9 +25,11 @@ def get_todo(db: Session, item_id: int):
     return db.query(ToDo).filter(ToDo.id == item_id).first()
 
 
-def update_todo(db: Session, item_id: int, content: str):
+def update_todo(db: Session, item_id: int, content: str, notes: str = None):
     todo = get_todo(db, item_id)
     todo.content = content
+    if notes is not None:
+        todo.notes = notes
```

Comment:
The update function now handles the `notes` field properly. Consider adding a check to see if the todo exists before updating, and return None or raise an exception if it doesn't.